### PR TITLE
chore(): update husky script

### PR DESCRIPTION
**Pull request by Tony133**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Update husky script

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Review by micalevisk — CHANGES_REQUESTED** [review]
as we're using `lint-staged`, I believe that we should have a configuration entry for it on `package.json`. See https://github.com/nestjs/docs.nestjs.com/pull/2314

Also, I'm getting these warnings:

```
hint: The '.husky/pre-commit' hook was ignored because it's not set as executable.
hint: You can disable this warning with `git config advice.ignoredHook false`.
hint: The '.husky/commit-msg' hook was ignored because it's not set as executable.
hint: You can disable this warning with `git config advice.ignoredHook false`.
```

**Comment by Tony133** [comment]
Yeah, I added it now in the package.json

**Comment by micalevisk** [comment]
I believe you should do `chmod +x .husky/pre-commit && chmod +x .husky/commit-msg`

**Comment by Tony133** [comment]
Done! now it should be ok 👍

**Review by micalevisk — APPROVED** [review]
it worked now! :heart:

File: package.json [hunk 1]
```diff
@@ -97,5 +93,8 @@
     ],
     "sourceMap": true,
     "instrument": true
+  },
+  "lint-staged": {
+    "packages/**/*.{ts,json}": []
```
Comment (micalevisk) [inline package.json:98]:
```suggestion
    "**/*.{ts,json}": []
```

Comment (Tony133) [reply package.json]: Done! I have updated the `package.json`

**Pull request merged**